src/components/AddForm.js

```diff
@@ -11,10 +15,18 @@ function AddForm() {
 
   function handleSubmit(e) {
     e.preventDefault();
-    console.log(destinationInput);
-    console.log(activitiesInput);
-    console.log(restaurantsInput);
-    console.log(notesInput);
+    addTripsToLocalStorage({
+      destination: destinationInput,
+      activities: activitiesInput,
+      restaurants: restaurantsInput,
+      notes: notesInput,
+    });
+    const trips = getTripsFromLocalStorage();
+    console.log(trips);
```

Comment:
console log detected 👀

Reply:
@FriederikeM You are my "Adlerauge" of the console log! 🙏